workflows/dnaseq/dnaseq-standard-fastq.wdl

```diff
@@ -27,12 +28,13 @@ workflow dnaseq_standard_fastq_experimental {
         prefix: "Prefix for the BAM file. The extension `.bam` will be added."
         aligner: {
             description: "BWA aligner to use",
-            choices: ["mem", "aln"]
+            choices: ["mem", "aln"],
```

Comment:
I don't think it's being enforced yet, but #11 says this will have to be line split. Fine with me to leave it for now or fix it in this PR. Your call.